inst/shiny/colourGroupInput.R

```diff
@@ -0,0 +1,55 @@
+#'
+#'
+
+colourGroupInput = function(inputId) {
```

Comment:
I tried it that way at first and it didn't work. It only worked when I put it with the shiny code.